lib/utils/utils.js

```diff
@@ -431,6 +431,10 @@ async function runSpinner (message, fn) {
   return result
 }
 
+function toDirectoryName (extensionId = '') {
```

Comment:
I would make the function name more explicit
```suggestion
function getExtensionDirectoryName (extensionId = '') {
```